crates/router/src/db/address.rs

```diff
@@ -91,38 +92,88 @@ impl AddressInterface for Store {
 
 #[async_trait::async_trait]
 impl AddressInterface for MockDb {
+    #[allow(clippy::unwrap_used)]
     async fn find_address(
         &self,
         _address_id: &str,
```

Comment:
Please remove the underscores from all function parameters that are being used in this implementation. We typically prefix variables with underscores when they're not being accessed.